src/System.Runtime.WindowsRuntime.UI.Xaml/src/System.Runtime.WindowsRuntime.UI.Xaml.csproj

```diff
@@ -2,11 +2,16 @@
   <PropertyGroup>
     <AssemblyName>System.Runtime.WindowsRuntime.UI.Xaml</AssemblyName>
     <ProjectGuid>{263DA4F1-C3BC-4B43-98E7-9F38B419A131}</ProjectGuid>
-    <PackageTargetRuntime>win8</PackageTargetRuntime>
     <Configurations>uap-Windows_NT-Debug;uap-Windows_NT-Release</Configurations>
```

Comment:
you also need to update the Configurations element so that VS doesn't fall behind.